setup.py

```diff
@@ -25,10 +25,11 @@
     "google_auth >= 1.16.0, < 2dev",
     "google-api-python-client >= 1.12.8",
     "gcs-oauth2-boto-plugin >= 2.7",
-    "htcondor >= 8.9.11",
-    "urllib3 >= 1.25.6",
-    "bill-calculator-hep >= 0.1.2",
+    "htcondor >= 9.0.0",
     "structlog >= 21.1.0",
+    "requests >= 2.14.2",
+    "urllib3 >= 1.26.2",
+    "bill-calculator-hep >= 0.1.2",
```

Comment:
bill-calculator-help should be 0.1.4, Maria made some fixes to it (renamed the package). Bruno updated the imports but it seems we missed this.

Reply:
Pushed update.